.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: trusty
+language: csharp
+solution: Banshee.sln
+
+addons:
+  apt:
+    packages:
+      - libgtk3.0-cil
+      - libdbus2.0-cil
+      - libtaglib2.1-cil
+      - libmono-addins0.2-cil
+      - libdbus-glib2.0-cil
+      - libgudev3.0-cil
+      - libmono-zeroconf1.0-cil
+      - libgconf2.0-cil
+env:
+  - BUILD_CONFIGURATION=Release
+  - BUILD_CONFIGURATION=Debug
+
+script:
+  - msbuild /p:Configuration=$BUILD_CONFIGURATION Banshee.sln
```

Comment:
Not that I'm aware of. As you can see [here](https://travis-ci.org/Carbenium/banshee) it builds just fine.
For the `feature/lite` branch we might have to adjust some things, but I haven't had any chance to look at that yet.

>I'm pretty sure autotools works on Windows too though :P

Oh well. You never stop learning new things. Although I prefer the msbuild approch for .NET projects.
We might even consider switching to a NuGet based approach to get the dependencies fulfilled. But that needs some more thought.